internal/core/data/router.go

```diff
@@ -54,4 +57,5 @@ func LoadRestRoutes(r *mux.Router, dic *di.Container) {
 
 	r.Use(correlation.ManageHeader)
 	r.Use(correlation.LoggingMiddleware(container.LoggingClientFrom(dic.Get)))
+	r.Use(handlers.ProcessCORS(edgeXContainer.ConfigurationFrom(dic.Get).Service.CORSConfiguration))
```

Comment:
Why can't we just have this call also in bootstrap here where all services will get it automatically?
https://github.com/edgexfoundry/go-mod-bootstrap/blob/main/bootstrap/handlers/httpserver.go#L97

Otherwise you have to add this call to each Core/Support services LoadRestRoutes and to the SDKs.

Reply:
> you are right. I move it to [edgexfoundry/go-mod-bootstrap#288](https://github.com/edgexfoundry/go-mod-bootstrap/pull/288) and verified it works

@cloudxxx8 , Thanks! Approved and merged that PR.